developer_tests/preprocess/work/input_current.nml

```diff
@@ -0,0 +1,19 @@
+&preprocess_nml
+   overwrite_output        = .true.,
+   input_obs_def_mod_file  = '../../../observations/forward_operators/DEFAULT_obs_def_mod.F90',
+   output_obs_def_mod_file = '../../../observations/forward_operators/obs_def_mod.f90',
+   input_obs_qty_mod_file  = '../../../assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90',
+   output_obs_qty_mod_file = '../../../assimilation_code/modules/observations/obs_kind_mod.f90',
+   obs_type_files          = '../../../observations/forward_operators/obs_def_1d_state_mod.f90'
+   quantity_files          = '../../../assimilation_code/modules/observations/oned_quantities_mod.f90'
+/
+
+&utilities_nml
+   TERMLEVEL      = 1,
+   logfilename    = 'dart_log.out'
+   nmlfilename    = 'dart_log.nml'
+   module_details = .false.
+   print_debug    = .false.
+   write_nml      = 'file'
+/
+
```

Comment:
as far as I can tell, this file is a duplicate of input.nml .... can we remove it?

Reply:
you have to copy other testing namelists to input.nml for testing different options.  this version is for testing the updated values. if you remove it you have to make a backup copy of input.nml before testing by hand.

Reply:
Since input_current.nml is identical to input.nml ... I wonder if the intended input.nml got overwritten? I understand the need to
copy test input to input.nml ... but should there be two identical files? Furthermore, "input_current.nml" is not a character string in any file, so it is not automatically being used for testing.